wrappers/python/xraydb/xraydb.py

```diff
@@ -430,8 +430,29 @@ def xray_lines(self, element, initial_level=None, excitation_energy=None):
                                                r.initial_level, r.final_level)
         return out
 
+    def xray_line_strengths(self, element):
+        """
+        return the absolute line strength in cm^2/gr for all available lines
+
+        arguments
+        ---------
+        element:  atomic number, atomic symbol for element
+
+        """
+        out = {}
+        for label, eline in self.xray_lines(element).items():
+            try:
+                edge = self.xray_edge(element, eline.initial_level)
+                mu = self.mu_elam(element, [edge.absorption_edge-2,
+                                            edge.absorption_edge+2], kind='photo')
```

Comment:
Well, in principle you just need above edge and below edge.  I guess changing `+/-2` to `*(1+/-0.001)` might be more robust.